tasks/eth/031-op-mainnet-pectra-defense/.env

```diff
@@ -4,9 +4,9 @@ USE_MULTI_DELEGATECALL=true
 TENDERLY_GAS=20000000
 
 # These are the current safe nonces for the OP Mainnet at time of writing.
-SAFE_NONCE_0X5A0AAE59D09FCCBDDB6C6CCEB07B7279367C3D2A=10
-SAFE_NONCE_0X847B5C174615B1B7FDF770882256E2D3E95B9D92=16
-SAFE_NONCE_0XC2819DC788505AAC350142A7A707BF9D03E3BD03=18
+SAFE_NONCE_0X5A0AAE59D09FCCBDDB6C6CCEB07B7279367C3D2A=9
+SAFE_NONCE_0X847B5C174615B1B7FDF770882256E2D3E95B9D92=15
+SAFE_NONCE_0XC2819DC788505AAC350142A7A707BF9D03E3BD03=17
```

Comment:
I think these were right before actually, the 030 task in CI messed things up?
```
cast call 0x5A0AAE59D09FCCBDDB6C6CCEB07B7279367C3D2A "nonce()(uint)"
cast call 0x847B5C174615B1B7FDF770882256E2D3E95B9D92 "nonce()(uint)"
cast call 0xC2819DC788505AAC350142A7A707BF9D03E3BD03 "nonce()(uint)"
10
16
18
```

Reply:
OK. Let's see: 0d19097

Reply:
You can see with simulate-sequence here I have made this to check more easier: 
and you are able to see which nonce are causing issue from the current .env. 

![CleanShot 2025-03-04 at 21 47 58@2x](https://github.com/user-attachments/assets/82205ea4-d9ef-4de1-8418-ae8329df93c8)

Reply:
fixed